index.html

```diff
@@ -408,7 +408,7 @@ <h3 class="neutal-blue">Frameworks</h3>
                     </ul>
                 </section>
                 <section class="bordered-card centered-column bg-light-green scale-transition">
-                    <img class="illustration" src="./assets/icons/about/icon-skills.svg" alt="lightening">
+                    <img class="illustration" src="./assets/icons/about/icon-skills.svg" alt="skills icon">
```

Comment:
> this comment actually refers to line #455-461:

- [ ] Kindly use `label` tags with `for` and `id` attributes for form control elements like input `tags`, So that input elements can have semantic labels, making them more accessible.